clients/cli/src/print_tree.rs

```diff
@@ -0,0 +1,30 @@
+use crate::error::CliResult;
+use crate::utils::{account, config};
+use crate::{err, err_unexpected};
+use lockbook_core::{get_file_by_path, read_document, Error as CoreError, GetFileByPathError};
+use std::io;
+
+pub fn print(file_name: &str) -> CliResult<()> {
+    account()?;
+    let cfg = config()?;
+
+    let file_metadata = get_file_by_path(&cfg, file_name).map_err(|err| match err {
+        CoreError::UiError(GetFileByPathError::NoFileAtThatPath) => {
+            err!(FileNotFound(file_name.to_string()))
+        }
+        CoreError::Unexpected(msg) => err_unexpected!("{}", msg),
+    })?;
+
+    // Temp copy and paste solution. To be reworked
```

Comment:
Cool didn't realize it was at this stage of a WIP, I'll let you continue to iterate in peace to get to the point where you have something that's functional. And we can do the next round of feedback at that point.